ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -287,8 +287,10 @@
       msg: "Looking at oc cluster up command:  '{{ oc_cluster_up_cmd }}'"
 
   - name: Run oc cluster up to start the cluster
-    shell: "{{ oc_cluster_up_cmd }}"
+    shell: "{{ oc_cmd }} cluster down && {{ oc_cluster_up_cmd }}"
```

Comment:
It's to avoid running oc cluster down on the first attempt. I thought that check would take a bunch of time when there was nothing around.  Ran it locally, it's very quick.